src/standalone/Bucket/bucket_parameterizations.jl

```diff
@@ -55,21 +55,21 @@ a helper function which computes and returns the surface air density for the buc
 model.
 """
 function ClimaLSM.surface_air_density(
-    atmos::PrescribedAtmosphere,
+    atmos::PrescribedAtmosphere{FT, CA, DT},
     model::BucketModel,
     Y,
     p,
     t,
     T_sfc,
-)
+) where {FT <: AbstractFloat, CA, DT}
```

Comment:
what does {CA, DT} gain us?

Reply:
I don't think we need them, I'll remove them